watertap/unit_models/tests/test_gac.py

```diff
@@ -524,7 +535,16 @@ def gac_frame_multi(self):
                 "BGAN": 0.1,
             },
             charge={"BGCAT": 1, "BGAN": -2},
+            diffus_calculation=DiffusivityCalculation.HaydukLaudie,
+            molar_volume_data={("Liq", "TCE"): 9.81e-5},
+            diffusivity_data={
+                ("Liq", "BGSOL"): 1e-5,
```

Comment:
So the user could provide diffusivity data for neutral solutes to initialize essentially. Is that right?

Reply:
That'll be overwritten by the calculation method since the config above.  But if we set config as `none`, neutral species can get values from `diffusivity_dt`

Reply:
`diffusivity_dt`?

Reply:
I guess my point is the following should be clear to the user: 
(1) What exactly am I intended to enter when choosing this calculation method?
(2) What are the repercussions of not entering my data the "typical" way? For example, do I get a warning that my value will be overwritten?
(3) What was my error if I did enter something incorrectly?

I know we raise one exception already. Are there others? If not, should there be warnings so that the user is aware of what's happening? Before, things were more straight forward since we were just supplying values for our components as needed. Now, it could be a mix of supplied values and calculated, so there could be unintended consequences that we should make sure to cover.